ideascube/search/utils.py

```diff
@@ -12,7 +12,7 @@ def create_index_table(force=True):
         cursor_transient.execute("DROP TABLE IF EXISTS idx")
         cursor_transient.execute("CREATE VIRTUAL TABLE idx using "
                                  "FTS4(id, model, model_id, public, text, "
-                                 "lang, kind, tags)")
+                                 "lang, kind, tags, package_id)")
```

Comment:
I disagree here.
Maybe package_id is a wrong name here, I concede. But it have to be seen as "from where this content comes" and it seems legit to have this information in this table.
Of course, we could put it in tags, but :
- Why not put everything in tag then (lang, kind, text, public) ?
- Tag is somehow free form information. There is no mandatory information nor mandatory format attached to them. If a tag become mandatory... were sure it will be there and we can move it out of tags and give him a real place.
- It can mix a lot with all other tags (not only package name). We cannot differentiate a tag from the other, so we cannot know if a tag is a package name or not.

Package_id (or other name) is a mandatory field. We (will) always have this information, and we need this information to filter media (this is use for by the link from the card).
Of course, we can **also** put the package's name in the tags to allow free search. But this is another feature.